### PR TITLE
feat: ✨ Use new moderation lists

### PR DESCRIPTION
**Pull request by LeonNicholls**
Support new moderation lists:
- nsfw.json: track nsfw OBJKTs that don't have the metadata set
- under_review.json: flag accounts that need to be reviewed by moderation team
- ignore_uri.json: IPFS URIs to ignore for potential terms and conditions violations

### CODE REVIEW

File: src/components/feed-item/index.js [hunk 1]
```diff
@@ -20,6 +22,7 @@ export const FeedItem = (props) => {
             creator: props.creator_id,
             objkt: String(props.id),
             displayView: true,
+            blur: nsfwList.includes(props.id.toString())
```
Comment (melMass) [inline src/components/feed-item/index.js]: Not tested yet but just looking at the edits can't the blur classname be passed one level up, on the container?

File: src/pages/mint/index.js [hunk 2]
```diff
@@ -358,6 +358,14 @@ export const Mint = () => {
 
     const uri0 = `ipfs://${hashv0}`
     const uri1 = `ipfs://${hashv1}`
+
+    // Ignore IPFS URI's that are in the ignore list; they can be minted multiple times
+    const ignoreUriList = getIgnoreUriList()
+    if (ignoreUriList.includes(uri0) || ignoreUriList.includes(uri1)) {
+      console.log('ignore uri')
```
Comment (melMass) [inline src/pages/mint/index.js]: remaining console logs :)

Btw how is this list generated?

File: src/pages/objkt-display/index.js [hunk 3]
```diff
@@ -156,14 +163,13 @@ export const ObjktDisplay = () => {
...
-                  <div
-                    style={{
-                      color: 'white',
-                      background: 'black',
-                      textAlign: 'center',
-                    }}
-                  >
-                    Restricted OBJKT
+                  <div className={styles.restricted}>
+                    Restricted OBJKT. Contact the Teia moderators on <a href="https://discord.gg/TKeybhYhNe" target="_blank" rel="noreferrer">Discord</a> to resolve the status. See the <a href="https://github.com/teia-community/teia-docs/wiki/Core-Values-Code-of-Conduct-Terms-and-Conditions#3-terms-and-conditions---account-restrictions" target="_blank" rel="noreferrer">Teia Terms and Conditions</a>.
```
Comment (melMass) [inline src/pages/objkt-display/index.js]: Might be done in a separate PR, but if possible it would be better to use an "external" file for texts

File: src/pages/objkt-display/styles.module.scss [hunk 4]
```diff
@@ -119,124 +96,120 @@
...
-  .history__ed,
-  .history__price,
-  .history__date {
-    display: block;
-  }
-
-  @include respond-to('desktop') {
-    display: none;
-  }
+    display: flex;
```
Comment (melMass) [inline src/pages/objkt-display/styles.module.scss:180]: Did husky worked on your end? There is a lot of formatting edits

**Comment by melMass** [comment]
I added a few comments besides that it seems all good, I will test it later 👍

File: src/pages/mint/index.js (hunk 2 above)
Comment (LeonNicholls) [reply src/pages/mint/index.js]: I thought in this case having something in the console for this case might be useful.
This list is manually updated with IPFS URI's that we want to ignore.

File: src/pages/objkt-display/index.js (hunk 3 above)
Comment (LeonNicholls) [reply src/pages/objkt-display/index.js]: Agreed. Seems like a lot of text in the GUI at the moment.

File: src/pages/objkt-display/styles.module.scss (hunk 4 above)
Comment (LeonNicholls) [reply src/pages/objkt-display/styles.module.scss:180]: Wasn't aware of the husky step. Don't think I did anything like that before. What do I do now?

File: src/components/feed-item/index.js (hunk 1 above)
Comment (LeonNicholls) [reply src/components/feed-item/index.js]: Not exactly sure what you mean by that.

Comment (melMass) [reply src/components/feed-item/index.js]: Instead of passing down `blur` 2/3 levels, just adding the classname to the container in this file

File: src/pages/objkt-display/styles.module.scss (hunk 4 above)
Comment (melMass) [reply src/pages/objkt-display/styles.module.scss:180]: It should autoinstall on npm install (the prepare script runs just after) the first time and its job is to run cmd on git events, we use it for commits, it lint/formats staged files to avoid this. I will check it maybe it actually needed those as it's only ran on staged files, but we ran it over the whole codebase once when we introduced husky so not sure.

Comment (LeonNicholls) [reply src/pages/objkt-display/styles.module.scss:180]: I'm not sure if it was automatically installed. Never saw any husky errors or anything that looked like a pre-commit trigger. This is what I got with the prepare target:
npm run prepare

> teia-ui@0.1.0 prepare
> husky install

husky - Git hooks installed

File: src/components/feed-item/index.js (hunk 1 above)
Comment (LeonNicholls) [reply src/components/feed-item/index.js]: Oh. Ok. Originally, I thought that each of the different kinds of media might have a different kind of "blur" and even different levels of blur; that is why I passed it down to each component. I ended up doing the same for each kind of media, but still not sure if that is the best to go with. I could imagine that some formats might not even do a CSS blur but black out the content.

File: src/pages/mint/index.js (hunk 2 above)
Comment (melMass) [reply src/pages/mint/index.js]: IMO there should be 0 log on a website, console.debug is meant for that if really needed.
After the refacto I wanted to convert every logs on the site (removing the useless ones)

File: src/pages/objkt-display/styles.module.scss (hunk 4 above)
Comment (melMass) [reply src/pages/objkt-display/styles.module.scss:180]: This should mean it's properly running, I'll test locally from main making the same edits to see the behaviour on my end

File: src/components/feed-item/index.js (hunk 1 above)
Comment (melMass) [reply src/components/feed-item/index.js]: This can be done with css only I think by using a class per media or something like that, I just feel that a visual feature should not be passed down that much components, maintenance wise it's not ideal

File: src/pages/mint/index.js (hunk 2 above)
Comment (LeonNicholls) [reply src/pages/mint/index.js]: Will remove.

File: src/components/feed-item/index.js (hunk 1 above)
Comment (LeonNicholls) [reply src/components/feed-item/index.js]: Ok. I thought that "displayView' was already an example of a visual feature being passed down, so I followed that pattern. But I will change to apply the blur to the top level element since they end up being a CSS filter for all of them (at least for now).

File: src/pages/objkt-display/styles.module.scss (hunk 4 above)
Comment (LeonNicholls) [reply src/pages/objkt-display/styles.module.scss:180]: Interesting. Just did a commit and now I'm seeing this (don't think I've seen this before):
ubuntupc:/hen/teia-ui$ git commit -m "Move blur to top level"
🔍  Finding changed files since git revision f1dd7672.
🎯  Found 22 changed files.
✍️  Fixing up src/components/feed-item/index.js.
✍️  Fixing up src/components/feed-item/styles.module.scss.
✍️  Fixing up src/components/media-types/video/style.css.
✍️  Fixing up src/pages/display/index.js.
✍️  Fixing up src/pages/display/styles.module.scss.
✍️  Fixing up src/pages/mint/index.js.
✅  Everything is awesome!
[new-moderation-lists 96b80887] Move blur to top level
 23 files changed, 625 insertions(+), 590 deletions(-)
 rewrite src/components/media-types/audio/styles.module.scss (84%)
 rewrite src/components/media-types/glb/styles.module.scss (85%)
 rewrite src/components/media-types/html/styles.module.scss (82%)
 rewrite src/components/media-types/image/styles.module.scss (74%)
 delete mode 100644 src/components/media-types/md/styles.module.scss
 rewrite src/components/media-types/pdf/styles.module.scss (78%)
 rewrite src/components/media-types/vector/styles.module.scss (78%)
 rewrite src/pages/display/styles.module.scss (78%)

**Comment by melMass** [comment]
I know I said it a few times already @jagracar but I think the last commit solves #110!
I'm going to merge it now, to see on the staging url

**Review by melMass — APPROVED** [review]
Great!
